graylog2-server/pom.xml

```diff
@@ -169,8 +169,7 @@
         <dependency>
             <!-- Workaround for https://github.com/FasterXML/jackson-bom/pull/38 -->
             <groupId>com.fasterxml.jackson.module</groupId>
-            <artifactId>jackson-module-scala_2.13</artifactId>
-            <version>2.9.10</version>
+            <artifactId>jackson-module-scala_3</artifactId>
```

Comment:
hmm, we haven't updated to scala 3 yet.
This workaround was only needed for jackson 2.9
I think it can be removed entirely